index.html

```diff
@@ -36,44 +38,45 @@ <h3>jackwoodpearce@gmail.com</h3>
         <article>
             <div class="img-holder"><img src="assets/pilot-shop.png" alt="pilot shop screen shot"></div>
             <div class="txt-holder">
-                <h4>The Pilot Shop</h4>
+                <h3>The Pilot Shop</h3>
                 <p>This was a two day build of a shop home page to enhance our skill to produce a responsive website.</p>
             </div>
         </article>
         <article>
             <div class="img-holder"><img src="assets/mayden-logo.jpg" alt="mayden academy logo"></div>
             <div class="txt-holder">
-                <h4>Mayden Logo HTML & CSS Only Challenge</h4>
-                <p>This was a two day build of a shop home page to enhance our skill to produce a responsive website.</p>
+                <h3>Mayden Logo HTML & CSS Only</h3>
```

Comment:
I think you might need to escape & with `&amp;`

Reply:
Nice spot. i'll change this